### PR TITLE
Enabled Third Party Invoicing

### PR DESCRIPTION
**Pull request by jaypatel512**
- Created samples.
- Added documentation.
- Fixed unit tests.
- Updated README

### CODE REVIEW

File: paypalrestsdk/api.py [hunk 1]
```diff
@@ -250,15 +250,15 @@ def headers(self, refresh_token=None, headers=None):
             "User-Agent": self.user_agent
         }
```
Comment (jaypatel512) [inline paypalrestsdk/api.py]: Now all API operations can pass in the refresh token. Added default to none for backward compatibility.

File: paypalrestsdk/invoices.py [hunk 2]
```diff
@@ -15,8 +15,8 @@ class Invoice(List, Find, Create, Delete, Update, Post):
     """
     path = "v1/invoicing/invoices"
 
-    def send(self):
-        return self.post('send', {}, self)
+    def send(self, refreshToken=None):
```
Comment (avidas) [inline paypalrestsdk/invoices.py]: In Python, the convention is to use underscore rather than camelcase so `refresh_token` :)

**Comment by avidas** [comment]
Other than one small comment LGTM ⚡

**Comment by jaypatel512** [comment]
Thanks @avidas for the code review.

**Pull request merged**